standalone/README.md

```diff
@@ -490,7 +490,17 @@ The list of all supported keys:
 * `endpoint`: The endpoint of the object store, e.g: https://s3.openshift-storage.svc:443 - **Optional**
 * `region`: The region of the object store - **Optional**
 
-A similar operation can be performed for the evaluation judge model serving service. Currently, the script expects the Judge serving service to be running and accessible from within the cluster. If it is not present, the script will not create this resource.
+### Judge Model Server Details
+
+A judge model is assumed to be running external to the script. This is used for model evaluation.
+
+The simplest way to provide details about a judge model server is with a Kubernetes secret. An example of how to serve a judge model can be found in
+the [kubernetes-yaml](../kubernetes_yaml) folder. This folder contains an example for serving a model and also for downloading a model to an S3
```

Comment:
Are we sure we want to point at this directory? (What if this moves to another repo?) Can we link to the `ServingRuntime` doc instead?

Reply:
where's that doc? if it has the same info, sure

Reply:
I'm fine with keeping the reference here for now & updating this with a follow-up

Reply:
Here I guess https://kserve.github.io/website/0.8/modelserving/servingruntimes/